visualization/app/codeCharta/ui/attributeSideBar/attributeSideBar.component.spec.ts

```diff
@@ -61,22 +61,22 @@ describe("AttributeSideBarComponent", () => {
 		expect(isSideBarOpen).toBe(true)
 
 		const name = container.querySelector("a[class=node-link]").textContent
-		await expect(name).toMatch("root/big leaf.ts")
+		expect(name).toMatch("root/big leaf.ts")
```

Comment:
The test cases are async functions. So we also would have to change the `it` cases. However, I don't think that this is a good solution. IntelliJ should be able to resolve the right method that is actually called.

Reply:
I am sorry but I think you are wrong. That's all jest synchronous 'toMatch'. 
![image](https://user-images.githubusercontent.com/72517530/194348799-2d23004d-915c-488f-91c3-0796211ee505.png)